models/management.cattle.io.globalrolebinding.js

```diff
@@ -23,4 +23,37 @@ export default {
     return this.user;
   },
 
+  basicNorman() {
+    if (this.id) {
+      return this.$dispatch(`rancher/find`, { id: this.id, type: NORMAN.GLOBAL_ROLE_BINDING }, { root: true });
+    }
+
+    return this.$dispatch(`rancher/create`, { type: NORMAN.GLOBAL_ROLE_BINDING, name: this.displayName }, { root: true });
+  },
+
+  async norman() {
+    const norman = await this.basicNorman;
+
+    norman.globalRoleId = this.globalRoleName;
+    norman.userId = this.userName;
```

Comment:
When saving bindings to a group (`/c/local/auth/group.principal/assign-edit?mode=edit`) i see 7 quick POST requests to `/v3/globalrolebindings`, all with 422 `must contain field [groupPrincipalId] OR field [userId]` errors.

Reply:
I resolved this issue. Unfortunately it uncovered an issue with removing roleTemplateBindings. After removing from bindings from an existing group they still showed up on the list page. I don't want to resolve that as part of this issue because it seems more involved and possibly an issue with management resources in general since this issue also came in https://github.com/rancher/dashboard/issues/3828.

In regards to YAML.. if I'm understanding correctly it sounds like we only need to block edit as yaml for non-rke2 cluster resources. Since these are all global resources I don't think I need to disable edit as yaml. Let me know if my understanding is incorrect.